addons/account/models/account_journal.py

```diff
@@ -542,6 +498,14 @@ def copy(self, default=None):
         return super(AccountJournal, self).copy(default)
 
     def write(self, vals):
+        # for journals, force a readable name instead of a sanitized name e.g. non ascii in journal names
+        if vals.get('alias_name') and 'type' not in vals:
+            self.ensure_one()  # cannot set same name on several journals, unicity constraint
```

Comment:
Won't this be raised anyway by the mail.alias code? Better to fail early but the error will be less explicit I feel.